### ISSUE TITLE
Efficient counting of data points for filtered measurement

### ISSUE THREAD

**Issue by buraketmen** [influxdata/influxdb-client-python#363]
**Proposal:**

Need to find out how many data points are in the filtered measurement in order to run my query with pagination.

**Current behavior:**
When I run the following query for a measurement with 100k data points, the response latency is a little bit high. I wanna know the number of pages in the form of int(count/10) when the page length is 10 based on the number of data points returned.

```console
bucket, measurement, start, stop ="test", "test", "0", "now()"
query = f'from(bucket:\"{bucket}\")' \
              f' |> range(start: {start}, stop: {stop})' \
              f' |> filter(fn: (r) => r._measurement == \"{measurement}\")' \
              f' |> count()'
```

__Desired behavior:__
Finding out how many data points are in a measurement in an efficient way.

__Use case:__
I couldn't see any method other than the one I am using now in InfluxDBv2.1.1 docs.

**Comment by bednar** [comment]
Hi @buraketmen,

thanks for using our client.

How many time this query takes in InfluxDB’s UI?

Do you have installed `ciso8061`?

- https://github.com/influxdata/influxdb-client-python#installation

Regards

**Comment by buraketmen** [comment]
> Hi @buraketmen,
> 
> thanks for using our client.
> 
> How many time this query takes in InfluxDB’s UI?
> 
> Do you have installed `ciso8061`?
> 
> * https://github.com/influxdata/influxdb-client-python#installation
> 
> Regards

Hi @bednar 

It was completed at roughly the same time as the query made on the UI. I filtered with one attribute in the query and it is a bit faster now. I just installed ciso8061 but I haven't tried it yet.

But I realized that my main problem is not with `count()` function but with `sort()` function in flux. When I use, `sort(columns:["_time"], desc: true)` while getting 10 data points ( `|> limit(n: 10, offset: 0)` ) , the query takes 5-7 seconds for filtered measurement that has 100k rows and 32 attributes.
